README.adoc

```diff
@@ -1,35 +1,87 @@
-// Before publishing your new repository:
-// 1. Write the readme file
-// 2. Update the issues link in Contributing section in the readme file
-// 3. Update the discussion link in config.yml file in .github/ISSUE_TEMPLATE directory
+= Java Operating System Statistics (jos_01)
 
-= repo-template
-
-// Add a short description of your project. Tell what your project does and what it's used for.
-
-This is a template repository for Teragrep organization.
+This project will extract statistics from the operating system and running processes into Java objects.
+You can use this to easily get information such as memory usage about running processes and the operating system itself.
+The information provided is based on the Linux filesystem's `/proc` files.
 
 == Features
 
-// List your project's features
+- Can be used to check if a process is alive by using process ID.
+- Provides the statistics of OS or any running process by using your system's `/proc` files as the source of truth.
+- Provides high level methods that calculate OS or process performance statistics automatically, such as `residentSetSize()`.
+- Can be used to create Java objects representing the current state of a `/proc` file and get access to the file's values.
+
+This project can be used to easily create system or process monitoring solutions in Java applications.
 
 == Documentation
 
 See the official documentation on https://docs.teragrep.com[docs.teragrep.com].
 
 == Limitations
 
-// If your project has limitations, please list them. Otherwise remove this section.
+- Only supports Linux-based operating systems
+- Requires access to `/proc` files in order to function
 
-== How to [compile/use/implement]
+== How to use
 
-// add instructions how people can start to use your project
+=== Requirements
+- Java version 1.8
+- Maven
 
-== Contributing
+=== Installation
+Include the following dependency into your project's pom.xml file:
+
+    <dependency>
+      <groupId>com.teragrep</groupId>
+      <artifactId>jos_01</artifactId>
+    </dependency>
+
+=== Usage
+
+==== Operating system statistics
+Start by creating a LinuxOS object.
+Optionally provide an object implementing SysconfInterface to manually configure the Operating system CPU tick rate.
```

Comment:
"Optionally provide an object BY implementing..."

If SysconfInterface is a parameter or anything else code related, turn it into `SysconfInterface`